web.py

```diff
@@ -171,6 +174,35 @@ async def root():
         </ul>
         """
 
+class FormattedFloat(float):
+    def __repr__(self):
+        return '{:.18f}'.format(self).rstrip('0')
+
+class JsonFloatEncoder(json.JSONEncoder):
+    def encode(self, val):
+        if isinstance(val, dict):
+            return {k: self.encode(v) for k, v in val.items()}
+
+        if isinstance(val, (list, tuple)):
+            return type(val)(self.encode(v) for v in val)
+
+        if isinstance(val, float):
+            return FormattedFloat(val)
+
+        return val
+
+class FormattedJSONResponse(JSONResponse):
+    media_type = "application/json"
+
+    def render(self, content: typing.Any) -> bytes:
+        return json.dumps(
+            str(content).replace("'", "\""),
+            cls = JsonFloatEncoder,
+            ensure_ascii=False,
```

Comment:
One of them is preferred. Instal pep8 and format the code.

Reply:
done